### ISSUE TITLE
Macular agenesis

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6797]
**Preferred term label:**
Macular agenesis

**Synonyms**

**Definition (free text, please give PubMed ID)**
A congenital defect characterized by lack of development of the macula.

**Parent term (use hpo.jax.org/app)**
Aplasia/Hypoplasia of the macula

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:106210 

PMID:7550230   1/9  

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
